test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -329,3 +343,53 @@ Describe "Import-Module -Force behaviour" -Tag "CI" {
         { Test-Two } | Should -Throw -ErrorId 'CommandNotFoundException'
     }
 }
+
+Describe "Get-Module -ListAvailable behaviour" -Tag "CI" {
+    BeforeAll {
+        $tempModulePath = Join-Path $TestDrive "TempModules"
+        $testModuleDir = Join-Path $tempModulePath "MyModuelTest"
+        $moduleManifest = Join-Path $testModuleDir "MyModuelTest.psd1"
+        $assemblyPath = Join-Path $testModuleDir "MyModuelTestCommandAssembly.dll"
+
+        $null = New-Item $testModuleDir -ItemType Directory -ErrorAction SilentlyContinue
+        if (-not (Test-Path $moduleManifest))
+        {
+            Set-Content $moduleManifest -Value @'
+@{
+    RootModule = 'MyModuelTestCommandAssembly.dll'
+    ModuleVersion = '0.0.1'
+    GUID = '5776ed43-1607-4e64-be76-acacdf8e9c8c'
+    FunctionsToExport = @()
+    CmdletsToExport = @("Get-Test")
+    AliasesToExport = @()
+}
+'@
+        }
+
+        $code = @'
+using System.Management.Automation;
+
+[Cmdlet("Get", "Test")]
+public class MyModuelTestCommand : PSCmdlet
+{
+    protected override void ProcessRecord()
+    {
+        WriteObject("BLAH");
+    }
+}
+'@
+        if (-not (Test-Path $assemblyPath))
+        {
+            Add-Type -TypeDefinition $code -OutputAssembly $assemblyPath
+        }
+    }
+
+    AfterAll {
+        Remove-Item -Path $tempModulePath -Recurse -Force
```

Comment:
This seems unnecessary, the test drive will be destroyed after the `Describe`.

Reply:
Yes, removed.